### DISCUSSION TITLE
How do I Chromaprint only those that aren't recognized by the autotagger?

### DISCUSSION THREAD

**Discussion by zaptrem** [beetbox/beets#4671] [Q&A]
I have a massive library to import and only want to run Chromaprint on those who are unable to be matched by the normal metadata matcher. How do I do this?

Thanks!

**Comment by wisp3rwind** [comment]
I suppose you could use the incremental import feature (see the `-i` flag: https://beets.readthedocs.io/en/stable/reference/cli.html#import): Do a first import run with the chroma plugin disabled, and skip all albums that are not matched. Then enable the plugin, and import the same directory again, which should skip everything that's already imported.
